tests/phpunit/tests/AMP/Optimization.php

```diff
@@ -99,6 +101,8 @@ public function test_get_optimizer_configuration_no_ssr() {
 
 		$transformers = [
 			AmpBoilerplate::class,
+			AmpBoilerplateErrorHandler::class,
```

Comment:
This is a somewhat unreleated changed, as the amp plugin was update on composer update.